content/python/concepts/strings/terms/isspace/isspace.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: '.isspace()'
+Description: 'Checks if all the characters in a string are whitespace characters.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+  - 'paths/data-science'
+---
+
+The **`.isspace()`** string method takes in a string and returns `True` if all the characters in a string are whitespaces, otherwise `False`.
```

Comment:
```suggestion
The **`.isspace()`** string method takes in a string and returns `True` if the entire string is composed of whitespace characters, otherwise `False`.
```